### PR TITLE
Sync docstring of Sequential.fit_generator() with that of improved Model.fit_generator()

### PR DESCRIPTION
**Pull request by ozabluda**

### CODE REVIEW

**Comment by ozabluda** [comment]
close/open to restart the tests (Travis seems to be up again)

File: keras/engine/training.py [hunk 1]
```diff
@@ -1969,7 +1969,7 @@ def fit_generator(self,
                       use_multiprocessing=False,
                       shuffle=True,
                       initial_epoch=0):
-        """Trains the model on data yielded batch-by-batch by a Python generator.
+        """Trains the model on data generated batch-by-batch by a Python generator.
```
Comment (Dref360) [inline keras/engine/training.py]: Sequence are not generators. So maybe 'Data generator' would be more suitable?

**Comment by ozabluda** [comment]
Conflicts resolved. On one hand, the actual commit is empty, so there is no need to review it. On the other had, this leaves a mystery why there was a conflict to begin with.

**Comment by Dref360** [comment]
There's still a comment to address at line 1972.

File: keras/engine/training.py (hunk 1 above)
Comment (ozabluda) [reply keras/engine/training.py]: I'd like to preserve phrasing "Python generator". Maybe "Python generator or an instance of `Sequence`"? That also corresponds to the docstring for the `generator` parameter:
>        # Arguments
>            generator: A generator or an instance of `Sequence`

https://github.com/keras-team/keras/blob/master/keras/engine/training.py#L1976

Comment (Dref360) [reply keras/engine/training.py]: Sounds good

**Comment by ozabluda** [comment]
Review comments addressed.

**Pull request merged**